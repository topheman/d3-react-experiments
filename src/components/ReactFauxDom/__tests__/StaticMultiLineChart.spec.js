import React from 'react';
import StaticMultiLineChart from '../StaticMultiLineChart/StaticMultiLineChart';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { mockDataLifeExpectancy } from '../../../test/jestHelpers';

describe('components/ReactFauxDom/StaticMultiLineChart', () => {
  it('should render correctly [shallow]', () => {
    const wrapper = shallow(
      <StaticMultiLineChart
        {...mockDataLifeExpectancy()}
      />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
