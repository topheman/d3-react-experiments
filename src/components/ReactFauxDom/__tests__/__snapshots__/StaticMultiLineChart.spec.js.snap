// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/ReactFauxDom/StaticMultiLineChart should render correctly [shallow] 1`] = `
<svg
  height={400}
  style={Object {}}
  width={700}
>
  <g
    style={Object {}}
    transform="translate(50,20)"
  >
    <path
      d="M345.69230769230774,227.27688564476887L378,217.0681265206813L410.3076923076923,206.42335766423358L432.9230769230769,167.39537712895378L436.15384615384613,167.39537712895378L439.38461538461536,167.39537712895378L442.6153846153846,167.39537712895378L445.84615384615387,167.39537712895378L449.0769230769231,158.39841849148416L452.3076923076923,158.39841849148416L455.53846153846155,158.39841849148416L458.7692307692308,158.39841849148416L461.99999999999994,158.39841849148416L465.2307692307693,148.669099756691L468.46153846153845,148.669099756691L471.6923076923077,148.669099756691L474.9230769230769,148.669099756691L478.15384615384613,148.669099756691L481.38461538461536,139.38686131386862L484.61538461538464,139.38686131386862L487.84615384615387,139.38686131386862L491.0769230769231,139.38686131386862L494.3076923076923,139.38686131386862L497.53846153846155,130.79014598540147L500.7692307692307,130.79014598540147L504,130.79014598540147L507.2307692307692,130.79014598540147L510.46153846153845,130.79014598540147L513.6923076923077,116.09610705596111L516.9230769230769,116.09610705596111L520.1538461538461,116.09610705596111L523.3846153846154,116.09610705596111L526.6153846153846,116.09610705596111L529.8461538461538,87.79379562043795L533.0769230769231,87.79379562043795L536.3076923076923,87.79379562043795L539.5384615384615,87.79379562043795L542.7692307692308,87.79379562043795L546,69.54014598540152L549.2307692307693,69.54014598540152L552.4615384615385,69.54014598540152L555.6923076923076,69.54014598540152L558.9230769230769,69.54014598540152L562.1538461538462,63.71107055961073L565.3846153846154,63.71107055961073L568.6153846153846,63.71107055961073L571.8461538461538,63.71107055961073L575.0769230769231,63.71107055961073L578.3076923076923,58.486618004866216L581.5384615384615,58.486618004866216L584.7692307692308,58.486618004866216L588,58.486618004866216L591.2307692307693,58.486618004866216L594.4615384615385,54.4884428223844L597.6923076923076,54.4884428223844L600.9230769230769,54.4884428223844L604.1538461538462,54.4884428223844L607.3846153846154,54.4884428223844L610.6153846153846,50.83515815085161L613.8461538461538,50.83515815085161L617.0769230769231,50.83515815085161L620.3076923076924,50.83515815085161L623.5384615384615,50.83515815085161"
      style={
        Object {
          "fill": "none",
          "stroke": "#87a9c5",
          "strokeWidth": "2px",
        }
      }
    />
    <path
      d="M432.9230769230769,204.89051094890513L436.15384615384613,204.89051094890513L439.38461538461536,204.89051094890513L442.6153846153846,204.89051094890513L445.84615384615387,204.89051094890513L449.0769230769231,193.84124087591243L452.3076923076923,193.84124087591243L455.53846153846155,193.84124087591243L458.7692307692308,193.84124087591243L461.99999999999994,193.84124087591243L465.2307692307693,179.34306569343067L468.46153846153845,179.34306569343067L471.6923076923077,179.34306569343067L474.9230769230769,179.34306569343067L478.15384615384613,179.34306569343067L481.38461538461536,170.55474452554748L484.61538461538464,170.55474452554748L487.84615384615387,170.55474452554748L491.0769230769231,170.55474452554748L494.3076923076923,170.55474452554748L497.53846153846155,164.65328467153284L500.7692307692307,164.65328467153284L504,164.65328467153284L507.2307692307692,164.65328467153284L510.46153846153845,164.65328467153284L513.6923076923077,161.566301703163L516.9230769230769,161.566301703163L520.1538461538461,161.566301703163L523.3846153846154,161.566301703163L526.6153846153846,161.566301703163L529.8461538461538,164.69586374695862L533.0769230769231,164.69586374695862L536.3076923076923,164.69586374695862L539.5384615384615,164.69586374695862L542.7692307692308,164.69586374695862L546,153.89355231143554L549.2307692307693,153.89355231143554L552.4615384615385,153.89355231143554L555.6923076923076,153.89355231143554L558.9230769230769,153.89355231143554L562.1538461538462,145.96958637469587L565.3846153846154,145.96958637469587L568.6153846153846,145.96958637469587L571.8461538461538,145.96958637469587L575.0769230769231,145.96958637469587L578.3076923076923,133.63868613138686L581.5384615384615,133.63868613138686L584.7692307692308,133.63868613138686L588,133.63868613138686L591.2307692307693,133.63868613138686L594.4615384615385,119.93248175182481L597.6923076923076,119.93248175182481L600.9230769230769,119.93248175182481L604.1538461538462,119.93248175182481L607.3846153846154,119.93248175182481L610.6153846153846,97.68065693430657L613.8461538461538,97.68065693430657L617.0769230769231,97.68065693430657L620.3076923076924,97.68065693430657L623.5384615384615,97.68065693430657"
      style={
        Object {
          "fill": "none",
          "stroke": "#401f93",
          "strokeWidth": "2px",
        }
      }
    />
    <path
      d="M0,179.47080291970806L3.230769230769231,183.17518248175182L6.461538461538462,185.85766423357666L9.692307692307693,191.26520681265205L12.923076923076923,182.96228710462287L16.153846153846153,180.36496350364962L19.384615384615387,186.8795620437956L22.615384615384613,179.72627737226276L25.846153846153847,183.26034063260343L29.07692307692308,185.9428223844282L32.30769230769231,189.22141119221413L35.53846153846154,182.4087591240876L38.769230769230774,190.0729927007299L42,182.23844282238443L45.230769230769226,181.4294403892944L48.46153846153847,181.0036496350365L51.69230769230769,197.43917274939176L54.92307692307692,184.6228710462287L58.15384615384616,200.8029197080292L61.38461538461539,181.89781021897812L64.61538461538461,171.33819951338202L67.84615384615385,182.32360097323598L71.07692307692308,182.53649635036498L74.3076923076923,175.76642335766422L77.53846153846155,178.06569343065692L80.76923076923076,176.78832116788323L84,179.72627737226276L87.23076923076924,176.4476885644769L90.46153846153845,169.8905109489051L93.6923076923077,164.44038929440393L96.92307692307693,176.8734793187348L100.15384615384615,177.89537712895378L103.38461538461539,178.44890510948906L106.61538461538463,197.05596107055962L109.84615384615384,165.71776155717762L113.07692307692308,172.14720194647202L116.30769230769232,174.1058394160584L119.53846153846153,168.9537712895377L122.76923076923077,196.161800486618L126,190.2007299270073L129.23076923076923,179.04501216545012L132.46153846153845,181.5571776155718L135.6923076923077,181.0888077858881L138.9230769230769,199.82360097323598L142.15384615384616,165.54744525547446L145.3846153846154,177.89537712895378L148.6153846153846,168.31508515815082L151.84615384615384,172.36009732360097L155.0769230769231,171.29562043795622L158.3076923076923,179.1727493917275L161.53846153846152,171.97688564476886L164.76923076923077,169.80535279805355L168,178.78953771289537L171.23076923076923,174.8722627737226L174.46153846153848,195.05474452554745L177.69230769230768,224.05109489051097L180.9230769230769,168.48540145985402L184.15384615384616,172.18978102189783L187.3846153846154,161.37469586374698L190.6153846153846,166.3990267639903L193.84615384615387,164.73844282238446L197.07692307692307,161.41727493917276L200.3076923076923,165.59002433090023L203.53846153846155,162.86496350364965L206.76923076923077,168.2299270072993L210,164.823600973236L213.23076923076925,166.56934306569343L216.46153846153845,166.01581508515815L219.69230769230768,169.33698296836985L222.92307692307693,163.0778588807786L226.15384615384616,166.48418491484185L229.3846153846154,163.97201946472023L232.61538461538464,162.35401459854015L235.84615384615384,156.30778588807786L239.07692307692307,165.54744525547446L242.30769230769232,162.35401459854015L245.53846153846155,164.73844282238446L248.76923076923077,164.73844282238446L252,156.47810218978103L255.23076923076923,157.84063260340633L258.46153846153845,147.7919708029197L261.6923076923077,146.21654501216548L264.9230769230769,154.47688564476886L268.15384615384613,157.75547445255475L271.3846153846154,158.35158150851584L274.61538461538464,150.17639902676402L277.8461538461538,145.79075425790757L281.0769230769231,144.04501216545015L284.3076923076923,145.492700729927L287.53846153846155,144.3430656934307L290.7692307692308,146.98296836982968L294,144.5985401459854L297.2307692307692,140.21289537712894L300.46153846153845,137.2749391727494L303.6923076923077,131.44160583941607L306.9230769230769,145.0669099756691L310.1538461538462,130.4197080291971L313.3846153846154,131.52676399026763L316.6153846153846,187.3479318734793L319.8461538461538,195.65085158150853L323.07692307692304,178.74695863746962L326.3076923076923,166.56934306569343L329.53846153846155,201.65450121654501L332.7692307692308,147.323600973236L336,130.59002433090023L339.2307692307692,125.94890510948906L342.46153846153845,116.45377128953771L345.69230769230774,117.56082725060827L348.92307692307696,114.92092457420924L352.1538461538462,118.66788321167886L355.38461538461536,120.20072992700733L358.6153846153846,112.66423357664235L361.8461538461538,114.19708029197082L365.0769230769231,119.09367396593677L368.3076923076923,108.02311435523114L371.53846153846155,107.76763990267642L374.7692307692308,106.36253041362534L378,104.48905109489056L381.2307692307692,101.67883211678833L384.4615384615385,101.80656934306572L387.69230769230774,99.72019464720194L390.9230769230769,98.18734793187349L394.15384615384613,98.99635036496352L397.38461538461536,96.27128953771287L400.6153846153846,139.02068126520686L403.84615384615387,104.48905109489056L407.0769230769231,105.55352798053528L410.3076923076923,122.92579075425792L413.53846153846155,149.02676399026765L416.7692307692308,116.0705596107056L420,84.5194647201946L423.23076923076917,77.7493917274939L426.4615384615385,69.82968369829689L429.6923076923077,73.70437956204381L432.9230769230769,67.40267639902675L436.15384615384613,68.55231143552317L439.38461538461536,63.05961070559613L442.6153846153846,63.357664233576656L445.84615384615387,59.653284671532845L449.0769230769231,58.58880778588809L452.3076923076923,58.46107055961073L455.53846153846155,56.6301703163017L458.7692307692308,51.478102189781055L461.99999999999994,51.26520681265208L465.2307692307693,50.37104622871044L468.46153846153845,47.901459854014604L471.6923076923077,49.94525547445255L474.9230769230769,50.58394160583947L478.15384615384613,46.49635036496352L481.38461538461536,47.2627737226278L484.61538461538464,45.51703163017032L487.84615384615387,45.55961070559613L491.0769230769231,45.602189781021934L494.3076923076923,46.7944038929441L497.53846153846155,43.00486618004868L500.7692307692307,43.1326034063261L504,42.06812652068129L507.2307692307692,41.42944038929443L510.46153846153845,39.981751824817536L513.6923076923077,39.598540145985396L516.9230769230769,38.57664233576645L520.1538461538461,36.06447688564475L523.3846153846154,35.55352798053531L526.6153846153846,34.361313868613195L529.8461538461538,33.8929440389295L533.0769230769231,33.211678832116775L536.3076923076923,31.636253041362522L539.5384615384615,31.76399026763994L542.7692307692308,29.67761557177613L546,29.336982968369853L549.2307692307693,28.18734793187349L552.4615384615385,25.760340632603402L555.6923076923076,24.695863746958594L558.9230769230769,24.05717761557179L562.1538461538462,22.822384428223813L565.3846153846154,22.055961070559647L568.6153846153846,20.863746958637478L571.8461538461538,20.608272506082756L575.0769230769231,18.81995133819953L578.3076923076923,18.39416058394164L581.5384615384615,17.585158150851612L584.7692307692308,16.05231143552311L588,15.37104622871044L591.2307692307693,14.68978102189783L594.4615384615385,13.412408759124105L597.6923076923076,12.645985401459882L600.9230769230769,11.836982968369796L604.1538461538462,11.79440389294399L607.3846153846154,7.834549878345513L610.6153846153846,7.834549878345513L613.8461538461538,6.08880778588815L617.0769230769231,5.152068126520703L620.3076923076924,4.854014598540175L623.5384615384615,4.257907542579062L626.7692307692308,3.1508515815085616L630,0"
      style={
        Object {
          "fill": "none",
          "stroke": "#86612d",
          "strokeWidth": "2px",
        }
      }
    />
    <path
      d="M190.6153846153846,186.6240875912409L222.92307692307693,182.0681265206813L255.23076923076923,169.54987834549877L287.53846153846155,156.47810218978103L306.9230769230769,141.36253041362534L319.8461538461538,177.46958637469587L352.1538461538462,105.51094890510947L384.4615384615385,88.13868613138686L420,92.34805352798054L432.9230769230769,62.46350364963507L436.15384615384613,62.46350364963507L439.38461538461536,62.46350364963507L442.6153846153846,62.46350364963507L445.84615384615387,62.46350364963507L449.0769230769231,56.57055961070557L452.3076923076923,56.57055961070557L455.53846153846155,56.57055961070557L458.7692307692308,56.57055961070557L461.99999999999994,56.57055961070557L465.2307692307693,55.77858880778592L468.46153846153845,53.22384428223842L471.6923076923077,52.37226277372264L474.9230769230769,51.94647201946475L478.15384615384613,49.81751824817519L481.38461538461536,49.81751824817519L484.61538461538464,49.3917274939173L487.84615384615387,48.540145985401466L491.0769230769231,50.24330900243308L494.3076923076923,50.669099756691026L497.53846153846155,49.3917274939173L500.7692307692307,48.540145985401466L504,47.68856447688569L507.2307692307692,46.836982968369796L510.46153846153845,45.55961070559613L513.6923076923077,45.98540145985402L516.9230769230769,44.282238442822404L520.1538461538461,41.727493917274955L523.3846153846154,41.727493917274955L526.6153846153846,40.02433090024334L529.8461538461538,39.598540145985396L533.0769230769231,38.32116788321167L536.3076923076923,37.04379562043795L539.5384615384615,35.76642335766428L542.7692307692308,34.06326034063261L546,33.63746958637472L549.2307692307693,32.360097323600996L552.4615384615385,30.656934306569326L555.6923076923076,29.805352798053548L558.9230769230769,28.953771289537713L562.1538461538462,29.12408759124088L565.3846153846154,28.18734793187349L568.6153846153846,26.271289537712903L571.8461538461538,25.888077858880763L575.0769230769231,24.610705596107096L578.3076923076923,23.844282238442872L581.5384615384615,22.822384428223813L584.7692307692308,20.991484184914896L588,19.458637469586392L591.2307692307693,18.43673965936739L594.4615384615385,17.329683698296833L597.6923076923076,15.839416058394136L600.9230769230769,15.456204379562053L604.1538461538462,15.158150851581524L607.3846153846154,12.645985401459882L610.6153846153846,12.049878345498769L613.8461538461538,10.602189781021934L617.0769230769231,9.92092457420921L620.3076923076924,9.537712895377126L623.5384615384615,9.069343065693431L626.7692307692308,8.388077858880763L630,7.153284671532845"
      style={
        Object {
          "fill": "none",
          "stroke": "#2d4a86",
          "strokeWidth": "2px",
        }
      }
    />
    <path
      d="M210,241.66861313868614L242.30769230769232,246.67591240875913L274.61538461538464,250.00048661800486L287.53846153846155,247.89537712895378L306.9230769230769,251.4447688564477L319.8461538461538,247.72506082725062L339.2307692307692,244.15523114355233L352.1538461538462,232.43917274939173L371.53846153846155,225.18199513381995L384.4615384615385,218.21776155717762L403.84615384615387,211.22287104622873L432.9230769230769,195.92761557177616L436.15384615384613,195.92761557177616L439.38461538461536,195.92761557177616L442.6153846153846,195.92761557177616L445.84615384615387,195.92761557177616L449.0769230769231,181.25912408759123L452.3076923076923,181.25912408759123L455.53846153846155,181.25912408759123L458.7692307692308,181.25912408759123L461.99999999999994,181.25912408759123L465.2307692307693,166.00304136253044L468.46153846153845,166.00304136253044L471.6923076923077,166.00304136253044L474.9230769230769,166.00304136253044L478.15384615384613,166.00304136253044L481.38461538461536,149.63138686131387L484.61538461538464,149.63138686131387L487.84615384615387,149.63138686131387L491.0769230769231,149.63138686131387L494.3076923076923,149.63138686131387L497.53846153846155,134.69464720194645L500.7692307692307,134.69464720194645L504,134.69464720194645L507.2307692307692,134.69464720194645L510.46153846153845,134.69464720194645L513.6923076923077,119.41727493917276L516.9230769230769,119.41727493917276L520.1538461538461,119.41727493917276L523.3846153846154,119.41727493917276L526.6153846153846,119.41727493917276L529.8461538461538,110.44586374695865L533.0769230769231,110.44586374695865L536.3076923076923,110.44586374695865L539.5384615384615,110.44586374695865L542.7692307692308,110.44586374695865L546,103.76094890510947L549.2307692307693,103.76094890510947L552.4615384615385,103.76094890510947L555.6923076923076,103.76094890510947L558.9230769230769,103.76094890510947L562.1538461538462,97.71046228710463L565.3846153846154,97.71046228710463L568.6153846153846,97.71046228710463L571.8461538461538,97.71046228710463L575.0769230769231,97.71046228710463L578.3076923076923,89.61618004866182L581.5384615384615,89.61618004866182L584.7692307692308,89.61618004866182L588,89.61618004866182L591.2307692307693,89.61618004866182L594.4615384615385,81.34732360097325L597.6923076923076,81.34732360097325L600.9230769230769,81.34732360097325L604.1538461538462,81.34732360097325L607.3846153846154,81.34732360097325L610.6153846153846,73.55535279805355L613.8461538461538,73.55535279805355L617.0769230769231,73.55535279805355L620.3076923076924,73.55535279805355L623.5384615384615,73.55535279805355"
      style={
        Object {
          "fill": "none",
          "stroke": "#81d279",
          "strokeWidth": "2px",
        }
      }
    />
    <g
      fill="none"
      fontFamily="sans-serif"
      fontSize={10}
      style={Object {}}
      textAnchor="middle"
      transform="translate(0,350)"
    >
      <path
        className="domain"
        d="M0.5,6V0.5H630.5V6"
        stroke="#000"
        style={Object {}}
      />
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(12.923076923076923,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          1,820
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(77.53846153846155,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          1,840
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(142.15384615384616,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          1,860
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(206.76923076923077,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          1,880
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(271.3846153846154,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          1,900
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(336,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          1,920
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(400.6153846153846,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          1,940
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(465.2307692307693,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          1,960
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(529.8461538461538,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          1,980
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(594.4615384615385,0)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x1={0.5}
          x2={0.5}
          y2={6}
        />
        <text
          dy="0.71em"
          fill="#000"
          style={Object {}}
          x={0.5}
          y={9}
        >
          2,000
        </text>
      </g>
    </g>
    <g
      fill="none"
      fontFamily="sans-serif"
      fontSize={10}
      style={Object {}}
      textAnchor="end"
    >
      <path
        className="domain"
        d="M-6,350.5H0.5V0.5H-6"
        stroke="#000"
        style={Object {}}
      />
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(0,350)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x2={-6}
          y1={0.5}
          y2={0.5}
        />
        <text
          dy="0.32em"
          fill="#000"
          style={Object {}}
          x={-9}
          y={0.5}
        >
          0
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(0,307.42092457420927)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x2={-6}
          y1={0.5}
          y2={0.5}
        />
        <text
          dy="0.32em"
          fill="#000"
          style={Object {}}
          x={-9}
          y={0.5}
        >
          10
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(0,264.84184914841853)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x2={-6}
          y1={0.5}
          y2={0.5}
        />
        <text
          dy="0.32em"
          fill="#000"
          style={Object {}}
          x={-9}
          y={0.5}
        >
          20
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(0,222.26277372262774)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x2={-6}
          y1={0.5}
          y2={0.5}
        />
        <text
          dy="0.32em"
          fill="#000"
          style={Object {}}
          x={-9}
          y={0.5}
        >
          30
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(0,179.683698296837)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x2={-6}
          y1={0.5}
          y2={0.5}
        />
        <text
          dy="0.32em"
          fill="#000"
          style={Object {}}
          x={-9}
          y={0.5}
        >
          40
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(0,137.10462287104622)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x2={-6}
          y1={0.5}
          y2={0.5}
        />
        <text
          dy="0.32em"
          fill="#000"
          style={Object {}}
          x={-9}
          y={0.5}
        >
          50
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(0,94.52554744525548)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x2={-6}
          y1={0.5}
          y2={0.5}
        />
        <text
          dy="0.32em"
          fill="#000"
          style={Object {}}
          x={-9}
          y={0.5}
        >
          60
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(0,51.94647201946475)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x2={-6}
          y1={0.5}
          y2={0.5}
        />
        <text
          dy="0.32em"
          fill="#000"
          style={Object {}}
          x={-9}
          y={0.5}
        >
          70
        </text>
      </g>
      <g
        className="tick"
        opacity={1}
        style={Object {}}
        transform="translate(0,9.367396593674016)"
      >
        <line
          stroke="#000"
          style={Object {}}
          x2={-6}
          y1={0.5}
          y2={0.5}
        />
        <text
          dy="0.32em"
          fill="#000"
          style={Object {}}
          x={-9}
          y={0.5}
        >
          80
        </text>
      </g>
    </g>
  </g>
</svg>
`;
